faqs/Tech/what-is-musd.mdx

```diff
@@ -0,0 +1,38 @@
+---
+id: what-is-musd
+sidebar_position: 2
+title: What is MUSD?
+---
+
+import CustomReactPlayer from "../../src/components/CustomReactPlayer";
+import RoundingError from "@site/static/img/faqs/roundingerror.png";
+
+MUSD is a stablecoin that allows users to borrow from the Moria Treasury, a contract on the Bitcoin Cash network. Users must provide, at a minimum 150%, collateral in BCH. MUSD is built on the Moria Protocol.
+
+The Moria Protocol is a decentralized stablecoin protocol built on top of the [CashTokens](what-is-cashtokens) system on Bitcoin Cash. It's currently being used to allow users to borrow the MUSD stablecoin. You can check out the [Moria Protocol White Paper Here.](/Moria-Whitepaper.pdf)
+
+## What is a stablecoin?
```

Comment:
-* Start by explaining that crypto is super volatile (I believe there's another FAQ to link to about that) and so people needed a solution.  Stablecoins were invented for that.  The idea is a crypto token that is redeemable against an asset (could be anything like you've said, but in most cases the USD).

Explain that a stablecoin flows across another network, not its own infrastructure. /faqs/Cryptocurrency/what-is-the-difference-between-utxo-and-account-model-chains#networks-vs-currencies

Then say the redemption method is different for various types of stablecoins and provide a description, quick explanation of the pros/cons of each method & example of each as a bullet listed:

## Redemption methods

there are various redemption methods

- **Company**: blah blah blah e.g. Tether, USDC by Coinbase
- **Algorithm**: blah blah blah e.g. DAI on ETH
- **Collateralised**: blah blah blah eg. MUSD

Reply:
Fixed.